make opera error 128

I download the go-opera master ZIP, unzip the file in Ubuntu 18.04. Running `make opera` produces the following:

```
GIT_COMMIT=`git rev-list -1 HEAD` && \
GIT_DATE=`git log -1 --date=short --pretty=format:%ct` && \
go build \
    -ldflags "-s -w -X github.com/Fantom-foundation/go-opera/cmd/opera/launcher.gitCommit=${GIT_COMMIT} -X github.com/Fantom-foundation/go-opera/cmd/opera/launcher.gitDate=${GIT_DATE}" \
    -o build/opera \
    ./cmd/opera
fatal: not a git repository (or any of the parent directories): .git
make: *** [Makefile:6: opera] Error 128
```

--- a/Makefile
+++ b/Makefile
@@ -3,8 +3,8 @@ all: opera
 
 .PHONY: opera
 opera:
-	GIT_COMMIT=`git rev-list -1 HEAD` && \
-	GIT_DATE=`git log -1 --date=short --pretty=format:%ct` && \
+	GIT_COMMIT=`git rev-list -1 HEAD 2>/dev/null || echo ""` && \
+	GIT_DATE=`git log -1 --date=short --pretty=format:%ct 2>/dev/null || echo ""` && \
 	go build \
 	    -ldflags "-s -w -X github.com/Fantom-foundation/go-opera/cmd/opera/launcher.gitCommit=$${GIT_COMMIT} -X github.com/Fantom-foundation/go-opera/cmd/opera/launcher.gitDate=$${GIT_DATE}" \
 	    -o build/opera \

